### ISSUE TITLE
Cloudera GCS connector parcel - DistCp guide

### ISSUE THREAD

**Issue by syvlabs** [GoogleCloudPlatform/professional-services#228]
Hi! This is just a suggestion, not really an issue -- please feel free to close this if this is not the right place to raise this.

**TL;DR:** it would be nice to show in the guide how to make the connector work for DistCp.

When we follow the guide, there is a issue that always pops up every time we do a DistCp transfer:
```
Error: java.lang.RuntimeException: java.lang.ClassNotFoundException: Class com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem not found
```
Which results from the `HADOOP_CLASSPATH` not being updated for all the worker nodes.

The README shows the following snippets for testing:
```
$ hdfs dfs -ls gs://bucket_name
```

However, we found out that this is not a good enough test for DistCp, since the `-ls` command does not need the `HADOOP_CLASSPATH` to be set in other worker nodes.

It would be very useful to have the Cloudera environment settings for DistCp in the guide, so it would be much more painless to onboard data engineers into GCP.

Thanks!

**Comment by syvlabs** [comment]
Tagging @yhqs540 again for assistance. Thank you!

**Comment by yhqs540** [comment]
I think I remember this thing. I am not 100% sure if there is a place in Cloudera Manager can be set to make effect on all workers. Will need to investigate.

**Comment by syvlabs** [comment]
We actually only solved this issue by sending the path to the JAR thru the `-libjars` option, although I don't think this is ideal. Would appreciate if there's a cleaner way to implement. Thanks!
